### ISSUE TITLE
Hora Smartdrive MX (A5-20-04) problem / typos in mapping

### ISSUE THREAD

**Issue by Brealiq** [mak-gitdev/HA_enoceanmqtt#121]
### The problem

Hi, I'm trying to use my Hora Smartdrive MX with EEP A5-20-04 properly using this addon. I wasn't able to send anything to the valve, I only could read the sensors.

After some time of getting used to the scripting of mapping.yaml and home assistant templating itself I found a few issues in the mapping.yaml. In line 3608 and line 3611 are typos, which prevented it to send any of those settings.

3608 (top is with typo):  
{% elif entity is search('config_BLC**S**',ignorecase=True) %}
{% elif entity is search('config_BLC',ignorecase=True) %}

3611:
{% set ns.SER = states(entity).split("-")[0]|int(default=0) %} **%}**
{% set ns.SER = states(entity).split("-")[0]|int(default=0) %}

And thank you for this great addon! I really like the Enocean protocol and it's awesome that you're actively maintaining it for home assistant.

### What type of installation are you running?

Addon

### What version of HA_enoceanmqtt has the issue?

 2024.1.3 

### What was the last working version of HA_enoceanmqtt?

 2024.1.3 

### What type of Home Assitant installation are you running?

Home Assistant Core

**Comment by mak-gitdev** [comment]
Hi @Brealiq,

Thanks for spotting these typos :+1:
Did you try to fix and if so is it working now ?

**Comment by mak-gitdev** [comment]
[Done](https://github.com/users/mak-gitdev/projects/1/views/1?pane=issue&itemId=50646835).
Will appear in 0.1.29

**Comment by Brealiq** [comment]
> Hi @Brealiq,
> 
> Thanks for spotting these typos 👍 Did you try to fix and if so is it working now ?

Yes, i added my customized mapping.yaml to the configuration and it is working now, I can set what I need.
I "just" need to create a template with the correct heating curve now.
